tensorflow/python/ops/summary_ops_v2.py

```diff
@@ -1205,14 +1205,13 @@ def trace_export(name, step=None, profiler_outdir=None):
       omitted, this defaults to `tf.summary.experimental.get_step()`, which must
       not be None.
     profiler_outdir: Output directory for profiler. It is required when profiler
-      is enabled when trace was started. Otherwise, it is ignored.
+      is enabled when trace was started. Otherwise, it is ignored. If you never
+      set it, the logdir set to the current SummaryWriter will be set.
```

Comment:
Maybe let's word this as

`Output directory for the profiler. This is only used when the profiler was enabled when the trace was started. In that case, if there is a logdir-based default SummaryWriter, this defaults to the same directory, but otherwise the argument must be passed.`

Reply:
Thank you for your advise! I fixed it! 🙇